those_names.py

```diff
@@ -0,0 +1,21 @@
+def full_names(first_names: list[str], last_names: list[str], min_length: int = 0) -> list[str]:
+    """
+    This function receive first and last names, make their first latter upper case and make all the combinations of the
+    names. If min_length received the names returns will be the names their length is >= to min_length.
+    :param first_names: List of the first names.
+    :param last_names: List of the last names.
+    :param min_length: Optional, the min_length of names.
+    :return: List of the names.
+    """
+    first_names = [name.title() for name in first_names]
+    last_names = [name.title() for name in last_names]
+    full_name = [first + " " + last for first in first_names for last in last_names]
```

Comment:
you can combine these three to one row, no need for all these loop. If you really want you can also put the length condition here too and save another loop.